pkg/webhook/overridepolicy/validating.go

```diff
@@ -46,10 +47,29 @@ func (v *ValidatingAdmission) Handle(_ context.Context, req admission.Request) a
 	}
 	klog.V(2).Infof("Validating OverridePolicy(%s/%s) for request: %s", policy.Namespace, policy.Name, req.Operation)
 
+	// Validate the FieldOverrider in Overriders
+	if err := validateFieldOverriders(&policy.Spec); err != nil {
+		klog.Error(err)
+		return admission.Denied(err.Error())
+	}
+
+	// Continue with existing validation
 	if errs := validation.ValidateOverrideSpec(&policy.Spec); len(errs) != 0 {
 		klog.Error(errs)
 		return admission.Denied(errs.ToAggregate().Error())
 	}
 
 	return admission.Allowed("")
 }
+
+// validateFieldOverriders ensures that either YAML or JSON is selected for each FieldOverrider.
+func validateFieldOverriders(overrideSpec *policyv1alpha1.OverrideSpec) error {
+	for _, overriderRule := range overrideSpec.OverrideRules {
```

Comment:
This verification code would fit better in the `pkg/util/validation/validation.go:ValidateOverrideRules` file.

Reply:
Edited.